spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -910,7 +907,15 @@ is still in progress using one of the following options:
   it exists, and if there is no such method it will reinvoke the `@Compensate`
   method). Please refer to the section
   <<reactive-support,about reactive support>> for more details.
-- A JAX-RS method can return a `202 Accepted` HTTP status code
+- A JAX-RS method can return a `202 Accepted` HTTP status code.
+  If there is no `@Status` method then the response MAY provide
+  a status URL in the `HTTP Location` header field so that the
+  implementation can discover the final outcome.
+  This URL, if present, MUST obey the requirements specificed in
+  the javadoc for the <<source-Status,Status annotation>>.
+  If the developer has not provided an `@Status` method nor a status
+  URL then the implementation may reinvoke the `@Compensate` method
+  (ie it MUST be idempotent).
```

Comment:
applies also for @Complete

Reply:
The text after the paragraph describing Compensate says:

> Similarly if the resource cannot perform a completion activity immediately.

Reply:
I think I've resolved everything here so @rdebusscher I will merge the PR at close of business today unless you have any further issues with it?